res/css/structures/_LeftPanel.pcss

```
@@ -28,6 +28,10 @@ Please see LICENSE files in the repository root for full details.
     --collapsedWidth: 68px;
 }
 
+.mx_LeftPanel_NewRoomList {
```

```suggestion
.mx_LeftPanel_newRoomList {
```

cf https://github.com/element-hq/element-web/blob/develop/code_style.md#stylesheets-pcss--postcss--plugins